src/main/resources/application.yml

```diff
@@ -8,8 +8,16 @@ spring:
       enabled: true
   jpa:
     hibernate:
-      ddl-auto: "create-drop" # FIXME: Just for early development without migrations
-
+      ddl-auto: "create" # FIXME: Just for early development without migrations
```

Comment:
Should we keep `create-drop` at least for now? If the entities change, this will fail.

Reply:
`create` recreates the database (drops existing tables first) on startup. create-drop (in addition) drops tables when application exits, so e.g. a failed deployment could wipe the database. I guess both are a bit problematic when there can be multiple instances of the app running.